src/js/package.json

```diff
@@ -17,7 +17,8 @@
     "postinstall": "npm run generate:serialization",
     "publish:wasm-api": "./publish-wasm-dependecy.js",
     "start:dev": "sudo npx ts-node ./modules/rpc/service.ts",
-    "start": "sudo node ./modules/rpc/service.js"
+    "start": "sudo node ./modules/rpc/service.js",
```

Comment:
this is interesting, why is sudo involved at all. I know unrelated to PR, but curious.

Reply:
Wasm engine needs to go into `/var/lib/redpanda/coprocessors` and when starting with `npm run start` your most likely a developer running with your own personal permissions. When we bundle and deploy `npm` isn't even involved. Maybe theres a more elegent way around this?